src/current/_includes/v24.1/ui/statement-details.md

```diff
@@ -30,7 +30,7 @@ The **Overview** section also displays the SQL statement fingerprint statistics
 |**Execution Retries** | The number of [retries]({{ link_prefix }}transactions.html#transaction-retries). |
 |**Execution Count** | The total number of executions. It is calculated as the sum of first attempts and retries. |
 |**Contention Time** | The amount of time spent waiting for resources. For more information about contention, see [Understanding and avoiding transaction contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention). |
-|**CPU Time** | The amount of CPU time spent executing the statement. The CPU time represents the time spent and work done within SQL execution operators. |
+|**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time nor KV execution time. |
```

Comment:
I could be wrong, but I think you would use "nor" if this were phrased like "It includes neither SQL planning time nor KV execution time." 

```suggestion
|**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time or KV execution time. |
```

Reply:
updated